__tests__/humanize.spec.js

```diff
@@ -45,10 +45,18 @@ describe('compactInteger tests', () => {
 
 describe('Ordinal value of numbers Test Suite', () => {
   describe('Ordinal value for numbers ending in zero', () => {
-    it('should return 0 if the number is 0 (cos 0th doesnt read very well)', () => {
+    it('should return 0 if the number is 0', () => {
       expect(Humanize.ordinal(0)).toEqual(0);
     });
 
+    it('should return the input if the number is 0', () => {
+      expect(Humanize.ordinal('0.')).toEqual('0.');
```

Comment:
I wonder if the method should be updated to return the string version of the parsed input rather than the original input. 

```js
expect(Humanize.ordinal(0)).toEqual('0');

// and 

expect(Humanize.ordinal('0.foobar')).toEqual('0');
```